tests/Command/ListConfigValuesCommandTest.php

```diff
@@ -102,17 +104,6 @@ public function testExecuteWithConnectionException()
             '/^Unable to connect to database./',
             $output
         );
-        $this->assertMatchesRegularExpression(
```

Comment:
Why pull these out of this test? They're part of the failure output.

Reply:
restored to the original behaviour.

:heavy_check_mark: